packages/modules/atlas-core/scripts/build.ts

```diff
@@ -40,6 +43,43 @@ async function main(): Promise<void> {
 
         await copyStylesAndAssets(mode === "start", outputDir);
     }
+
+    await buildAndCopyWidgets(mode, ["feedback-native"], outputDir);
+}
+
+async function buildAndCopyWidgets(command = "build", widgets: string[] = [], destination?: string) {
+    let cwd = process.cwd();
+    if (cwd.endsWith("atlas-core")) {
+        cwd = join(cwd, "../../../");
+    }
+    execSync(
+        `${
+            command !== "release" && destination ? `npx cross-env MX_PROJECT_PATH=${destination} ` : ""
+        }npm run ${command} -- --scope '${
+            widgets.length > 1 ? `{${widgets.join(",")}}` : widgets.join("")
+        }' --include-dependencies`,
+        {
+            stdio: "inherit",
+            cwd
+        }
+    );
+    if (command === "release") {
+        const pluggableWidgetsFolderPath = join(cwd, "packages/pluggableWidgets");
+        const mpkPathsToCopy = [];
+        for await (const widget of widgets) {
+            const version = require(join(pluggableWidgetsFolderPath, widget, "package.json")).version;
+            const widgetDistPath = join(pluggableWidgetsFolderPath, widget, "dist", version);
+            const distExists = existsSync(widgetDistPath);
+
+            if (distExists) {
+                mpkPathsToCopy.push(...(await readdir(widgetDistPath)).map(name => join(widgetDistPath, name)));
```

Comment:
Very nitty: If you use `concat` you don't have to spread 😄

Reply:
Concat sounds so ES5 😛